tailcfg/tailcfg.go

```diff
@@ -734,17 +734,21 @@ type NetInfo struct {
 	// the control plane.
 	DERPLatency map[string]float64 `json:",omitempty"`
 
+	// FirewallMode is the current firewall utility in use by router (iptables, nftables).
+	// If FirewallMode is empty, nftables is being used.
```

Comment:
"nftables" is given as an example, but then says empty means nftables. So both mean nftables? I assume only on Linux.

What about neither? (userspace mode)